index.js

```diff
@@ -3,18 +3,28 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 
 const makeEncryptionFormat = require('./format')
+const monitorForest = require('./monitor')
 
 exports.name = 'box2'
 
 exports.init = function (ssb, config) {
   const encryptionFormat = makeEncryptionFormat()
   if (ssb.db) ssb.db.installEncryptionFormat(encryptionFormat)
 
+  if (config.box2 && config.box2.legacyMode) {
```

Comment:
reminder: document this. In general there should probably be a section in the README that talks about these two modes but that probably is depends on the spec for this :)

Reply:
Documented